Makefile

```
@@ -186,16 +226,16 @@ clean: mostlyclean
 	if [ -r "$$FILE" ] ; then \
 		$(CMD_DOCKER) rmi "$$(< $$FILE)" ; \
 	fi
-	rm -rf $(OUT_DIR)
 	$(MAKE) -C $(LIBBPF_SRC) clean
-	$(MAKE) -C $(BPF_ROOT) clean
-
-check_%:
-	@command -v $* >/dev/null || (echo "missing required tool $*" ; false)
+	-rm -rf $(OUT_DIR)
```

looks like this hyphen should not be here

```suggestion
	rm -rf $(OUT_DIR)
```

---

This was one purpose. The prefix hyphen prevents make action from failing if this command fails.